src/System.Net.Http/tests/FunctionalTests/PostScenarioTest.cs

```diff
@@ -173,7 +173,23 @@ public PostScenarioTest(ITestOutputHelper output)
             var credential = new NetworkCredential(UserName, Password);
             await PostUsingAuthHelper(serverUri, ExpectedContent, content, credential, preAuthenticate: true);
         }
-        
+
+        [OuterLoop] // TODO: Issue #11345
+        [Theory, MemberData(nameof(EchoServers))]
+        public async Task PostNoContent_ExcludeContentTypeHeader_Success(Uri serverUri)
```

Comment:
>the naming scheme here 

This the book we recommend. The naming scheme comes from here:
https://smile.amazon.com/Art-Unit-Testing-examples/dp/1617290890/ref=sr_1_1?ie=UTF8&qid=1496771523&sr=8-1&keywords=art+of+unit+testing